src/components/Items/items.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Item from "./item/item";
+import { connect } from "react-redux";
+import { mark_checked, removeTodo} from "../../actions/actions";
+
+function Items(props){
+    let unchecked_items = [];
+    let checked_items = [];
+    const {items=[], filtered_items = []} = props;
+    const itemsToRender = filtered_items.length ? filtered_items : items;
+    
+    itemsToRender.forEach(item => {
```

Comment:
Modify this logic. Conditionally set the check prop. Do not create another array and then append it later.

Reply:
Cannot be done in any other way as discussed.